examples/sb3_imitation.py

```diff
@@ -0,0 +1,199 @@
+)
+parser.add_argument(
+    "--demo_files",
+    nargs="+",
+    type=str,
+    help="""One or more files with recoded expert demos, with a space in between, e.g. "demo1.json", demo2.json""",
+)
+parser.add_argument("--seed", type=int, default=0, help="seed of the experiment")
+parser.add_argument(
+    "--save_model_path",
+    default=None,
+    type=str,
+    help="The path to use for saving the trained sb3 model after training is complete. Extension will be set to .zip",
+)
+parser.add_argument(
+    "--onnx_export_path",
+    default=None,
+    type=str,
+    help="The Godot binary to use, do not include for in editor training",
+)
+parser.add_argument(
+    "--viz",
+    action="store_true",
+    help="If set, the simulation will be displayed in a window during training. Otherwise "
+    "training will run without rendering the simulation. This setting does not apply to in-editor training.",
+    default=False,
+)
+parser.add_argument(
+    "--run_eval_after_training",
+    action="store_true",
+    help="Evaluate policy in an env after training in a single env. Will always visualize.",
+    default=False,
+)
+parser.add_argument("--speedup", default=1, type=int, help="Whether to speed up the physics in the env")
+parser.add_argument(
+    "--n_parallel",
+    default=1,
+    type=int,
+    help="How many instances of the environment executable to " "launch - requires --env_path to be set if > 1.",
+)
+parser.add_argument(
+    "--il_timesteps",
+    default=0,
+    type=int,
+    help="How many timesteps to train for using imitation learning.",
+)
+parser.add_argument(
+    "--rl_timesteps",
+    default=0,
+    type=int,
+    help="[Optional] Additional timesteps to train for using RL after IL.",
+)
+parser.add_argument(
+    "--eval_episode_count",
+    default=0,
+    type=int,
+    help="[Optional] How many episodes to evaluate for after training.",
+)
+
+
+args, extras = parser.parse_known_args()
+
+
+def handle_onnx_export():
+    # Enforce the extension of onnx and zip when saving model to avoid potential conflicts in case of same name
+    # and extension used for both
+    if args.onnx_export_path is not None:
+        path_onnx = pathlib.Path(args.onnx_export_path).with_suffix(".onnx")
+        print("Exporting onnx to: " + os.path.abspath(path_onnx))
+        export_ppo_model_as_onnx(learner, str(path_onnx), use_obs_array=True)
+
+
+def handle_model_save():
+    if args.save_model_path is not None:
+        zip_save_path = pathlib.Path(args.save_model_path).with_suffix(".zip")
+        print("Saving model to: " + os.path.abspath(zip_save_path))
+        learner.save(zip_save_path)
+
+
+def close_env():
+    try:
+        print("closing env")
+        env.close()
+    except Exception as e:
+        print("Exception while closing env: ", e)
+
+
+trajectories = []
+
+for file_path in args.demo_files:
+    with open(file_path, "r") as file:
+        data = json.load(file)
+
+    for i in range(0, len(data)):
```

Comment:
this could be just `for traj in data`, then `obs=traj[0]` and `acts=traj[1]`

no problem if you wanna leave as is though